modules/toDoTasks.js

```diff
@@ -59,6 +58,20 @@ class ToDoTasks {
         }
       });
     });
+
+    const checkboxes = document.querySelectorAll('.checkbox');
+
+    checkboxes.forEach((checkbox) => {
+      checkbox.addEventListener('change', (event) => {
+        if (event.target.checked) {
+          this.markComplete(event);
+          // label.classList.toggle('checked');
+        } else {
+          this.markUnComplete(event);
+          // label.classList.toggle('checked');
```

Comment:
- [ ] Kindly remove the commented code as suggested by the previous reviewer. It does not make your code look tidy.